src/App.js

```diff
@@ -1,11 +1,15 @@
 import React from 'react';
 // eslint-disable-next-line import/extensions
```

Comment:
- [ ] [OPTIONAL]
You have **disabled ESLint checks** on parts of your code. Please enable **all** ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust and clean code.